README.md

```diff
@@ -120,17 +139,33 @@ Methods:
 | stroke linecap  | std::string | null             |
 | stroke linejoin | std::string | null             |
 
-| Method            | Parameter type | Description           | 
+| Method            | Parameter type | Description           |
 |-------------------|----------------|-----------------------|
 | SetStrokeColor    | svg::Point     | Sets stroke color.    |
 | SetFillColor      | svg::Point     | Sets fill color.      |
 | SetStrokeWidth    | float          | Sets stroke width.    |
 | SetStrokeLinecap  | std::string    | Sets stroke linecap.  |
 | SetStrokeLineJoin | std::string    | Sets stroke linejoin. |
 
+### svg::Section
+
+Section is an object type with unique properties. It doesn't have most properties, listed above and doesn't specify
```

Comment:
It doesn't have most properties listed above as it's not a figure.   
A Section is a set of objects, such as Text, Circle, Section etc.

Reply:
Done.